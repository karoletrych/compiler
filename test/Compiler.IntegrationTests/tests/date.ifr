module Date
fun main : void
{
	System::Console:.WriteLine(System::DateTime:.MinValue.ToString());
}
