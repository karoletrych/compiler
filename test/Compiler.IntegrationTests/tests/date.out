01.01.0001 00:00:00